app/models/speech.rb

```diff
@@ -0,0 +1,8 @@
+class Speech < ContentItem
+  include Body
```

Comment:
I think this commit should come later once you've committed the concerns included here.